metricbeat/module/kubernetes/_meta/test/docs/README.md

```diff
@@ -0,0 +1,157 @@
+
+Usually we should be ok with the kubernetes version that minikube creates, but you can force it by using `--kubernetes-version` flag.
+
+```
+minikube start --kubernetes-version v1.15.0
+```
+
+## Playground Pod
+
+A playground Pod hosts the ubuntu container metricbeat will be running. A working playground is provided under [./01_playground](./01_playground) subfolder.
+
+This file contains:
+
+- a service account.
+- a cluster role, if you are consuming kubernetes API resources, make sure that the APIGroup/Version, Resource and verb are listed here.
+- a cluster role binding that links the service account to the service role
+- an Ubuntu Pod:
+  - uses `hostNetwork`, so it can reach ports at the host instance (for instance, the kubelet)
+  - executes `sleep infinity`, so that it never exists, but does nothing
+  - in order to be useful for filebeat, it mounts `/var/log/`, `/var/lib/docker/containers` and `/var/lib/filebeat-data`
+
+At the time of writing this the Pod has been only used for 2 tests from the same person (hello), there is a lot of room for improvement.
+
+To deploy the pod _as is_ you need to:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/elastic/beats/master/metricbeat/module/kubernetes/_meta/test/docs/01_playground/playground-ubuntu.yaml
+```
+
+## Test
+
+
+Binary and assets needed for the test that we prepared above need to be copied to the playground pod. Use `kubectl` to copy the directory, further iterations might only need to copy the changing assets.
+
+Replace source folder and Pod namespace/name
+
+```
+kubectl cp --no-preserve  /home/myuser/playground/metricbeat playground:/metricbeat
+```
+
+Now you can exec into the container and launch metricbeat
+
+```
+ kubectl exec -ti playground /bin/bash
+
+ cd /metricbeat
+ ./metricbeat -c metricbeat.yml  -e
+
+ ```
+
+### Test Iterations
+
+When copying new assets to an already used playground Pod, you will most probably run into an issue:
+```
+tar: metricbeat/kibana/7/dashboard/Metricbeat-aerospike-overview.json: Cannot open: Not a directory
+tar: metricbeat/kibana/7/dashboard/Metricbeat-apache-overview.json: Cannot open: Not a directory
+tar: metricbeat/kibana/7/dashboard/Metricbeat-ceph-overview.json: Cannot open: Not a directory
+tar: metricbeat/kibana/7/dashboard/Metricbeat-consul-overview.json: Cannot open: Not a directory
+```
+
+I haven't looked much into this, there seems to be something going on when kubernetes untars the bundled directory. As a workaround, delete the metricbeat directory at the Pod before copying a new set of assets.
+
+# Testing kubernetes loads
+
+## Kube-state-metrics
+
+Kube-state-metrics needs to be deployed for all the `state_` prefix metricsets at kubernetes. Yamls are to be found at the [upstream project](https://github.com/kubernetes/kube-state-metrics/tree/master/kubernetes)
+
+For the sake of laziness they are copied under [02_kubestatemetrics](./02_kubestatemetrics)
```

Comment:
I think it'd be ok with mentioning here how to clone the `kube-stats-metrics` repo and use the yaml files there instead of copying them. And it'd be one thing less to maintain :slightly_smiling_face: